### ISSUE TITLE
Arrow -> accept disables right arrow navigation

### ISSUE THREAD

**Issue by SinanAkkoyun** [rjmacarthy/twinny#107]
**Describe the bug**
Hi, thank you very much for the awesome repo!
When getting FIM autosuggested, I have no other choice than to accept it, but in reality I just wanna navigate through my code with the arrow keys

**To Reproduce**
1. Go in the middle of a line
2. Try to move right with the arrow keys without accepting the suggestion

It would be super awesome to have a setting that disables that, as TAB accept is good enough, thanks!

**Comment by AntonKrug** [comment]
Could you explain more precisely how to reporoduce it?

When I trigger the auto complete i have `TAB` and `Ctrl + right arrow` to accept, right arrow for me doesn't accept:

![image](https://github.com/rjmacarthy/twinny/assets/30338596/07307ebb-0647-47aa-885a-4915e5c7ed2c)

Disclaimer: I'm not repo owner, so maybe i'm missing something obvious.

**Comment by SinanAkkoyun** [comment]
@AntonKrug You are completely right, I forgot to add that I quickly navigate with Ctrl + right arrow, not right arrow alone, that was my bad!

I mostly use ctrl right arrow to navigate per word, I would love to see an option to disable CTRL+RightArrow accepting

**Comment by rjmacarthy** [comment]
Hey, I think I this is possible in vscode key bindings?

**Comment by SinanAkkoyun** [comment]
Ohhh yes, thank you so much!
